index.html

```diff
@@ -0,0 +1,40 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@300&display=swap" rel="stylesheet">
+    <title>Priscilla Portfolio</title>
+</head>
+
+<body>
+    <div class="central-wrapper">
+        <header class="header-class">
+           <a href="#" target="_blank" rel="noopener noreferrer"><h3 class="logo-text">My Logo</h3></a>
+            <div class="hamburger">
+              <a href="#"><img src="resources/img/Union.png" alt=""></a> 
+            </div>
+        </header>
+        <main class="main-wrapper">
+            <h2 class="name">I am Priscilla OYEBADE, Good to have you here!</h2>
+            <p class="self-intro">I am a seasoned Software Developer, ready to work remotely with any Tech Organization.
+                All of my proffessional works can be accessed via GitHub. Kindly go through my repositories
+                to see the projects have done with colleague remotely.</p>
+            <p class="social-connect"> LET'S CONNECT</p>
+            <div class="social-media-icons">
+                <a href="#"><img class="media1" src="resources/img/Vector.png" alt="twitter"></a>
+                <a href="#"><img class="media2" src="resources/img/Linkedinicon.png" alt="linkedln"></a>
+                <a href="#"><img class="media3" src="resources/img/Vector1.png" alt="fact"></a>
+                <a href="#"><img class="media4" src="resources/img/Vector2.png" alt="github"></a>
+                <a href="#"><img class="media5" src="resources/img/Vector3.png" alt="amazon"></a>
+            </div>
```

Comment:
Kindly consider changing your `<div>` to a `<ul>` because it is a list, and your `<a>` tags inside `<li>` tags to complete the list :tada: this is considered a good practice! :+1: